javascript/builtins/Function.json

```
@@ -749,7 +749,7 @@
                 "opera": "mirror",
                 "opera_android": "mirror",
                 "safari": {
-                  "version_added": false
+                  "version_added": "18"
```

On https://compat-table.github.io/compat-table/es2016plus/ I see 6/7 tests pass from Safari 17 onwards. (I think the OP also mentioned 17).

```suggestion
                  "version_added": "17"
```

---

The one test that didn't pass is a question mark rather than a failure too, so I took the code and ran it manually in Safari 17.5.  Confirmed it works as expected, so it appears the WebKit commit's tags are slightly off!